pkgs/default.nix

```diff
@@ -10,6 +10,7 @@
   verilator_ot = import ./verilator {inherit pkgs;};
   python_ot = pkgs.callPackage ./python_ot {inherit inputs;};
 }
+// import ./llvm {inherit pkgs;}
```

Comment:
Perhaps rename llvm directory to llvm_cheriot and follow the same convention as the verilator_ot/python_ot above?

Reply:
That sounds sensible. I had visions of having multiple llvm toolchains, but we can create sub directories as they are needed.

It doesn't have any additional files, so created a file instead of a superfluous directory.